analytics/policy/services/registration_number_policies.py

```diff
@@ -0,0 +1,39 @@
+import re
+import json
+import urllib3
+
+class RegistrationNumberPolicies:
+    def __init__(self, registration_number):
+        self.registration_number = registration_number
```

Comment:
I think the registration number should be retrieved from listing model.
Please check analytics/models/listing.py

Reply:
Don't you want to use registration_number_poiclies as a helper to check the validity of the license number?
In this case Listing will be imported in another part of the code.

Reply:
I mean the attribute `registration_number` should be taken from Listing object
But this maybe fine for now for simplicity